editoast/src/modelsv2/timetable.rs

```diff
@@ -0,0 +1,76 @@
+use crate::diesel::query_dsl::methods::DistinctDsl;
+use crate::error::Result;
+use crate::models::List;
+use crate::models::NoParams;
+use crate::modelsv2::{Retrieve, Row};
+use crate::tables::timetable_v2::dsl;
+use crate::views::pagination::Paginate;
+use crate::views::pagination::PaginatedResponse;
+use async_trait::async_trait;
+use diesel::sql_query;
+use diesel::sql_types::{Array, BigInt, Nullable};
+use diesel_async::AsyncPgConnection as PgConnection;
+use editoast_derive::ModelV2;
+
+#[derive(Debug, Default, Clone, ModelV2)]
+#[model(table = crate::tables::timetable_v2)]
+#[model(changeset(public))]
```

Comment:
I'm not a fan of exposing the changeset's fields. I'd rather consider the cs's internals as opaque. What's the use case here?

Reply:
Good catch, it's not needed in the final version of this PR.